diktat-gradle-plugin/src/test/kotlin/org/cqfn/diktat/plugin/gradle/UtilsTest.kt

```diff
@@ -0,0 +1,24 @@
+package org.cqfn.diktat.plugin.gradle
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+class UtilsTest {
+    @Test
+    fun `test gradle version`() {
+        Assertions.assertEquals(
+            GradleVersion.fromString("6.6.1"),
```

Comment:
why do you need to test gradle version. It can be changed in any moment. No?

Reply:
I'm testing my `GradleVersion` class, which is used to determine which type of task is supported by current gradle.